### PR TITLE
Add sessions status filter

### PR DESCRIPTION
**Pull request by tosametal**
Add `StatusFilter` component to filter sessions by status.

The below is a sample screen.

### CODE REVIEW

**Comment by muga** [comment]
@tosametal Thanks! That's great. I will take a look.

**Comment by muga** [comment]
@tosametal the function and the impl logic looks good to me. one thing is that we may need to explain what is the drop-down filter. Can we put `Status: ` message on the left of the drop-down button? What do you think? 

@dzannotti if you get a chance, can I consult you about the look and feel of the change? 

I also think that we'd better to support pagination buttons or automatic scrolling for page loading but, that would be future improvement.

**Comment by tosametal** [comment]
@muga 
Thank you for reviewing.
I agree with your suggestion. I will add `Status: ` message.
Also, since I noticed that status name `Failed` should be `Failure`, I will fix it.

> I also think that we'd better to support pagination buttons or automatic scrolling for page loading but, that would be future improvement.

If I have a chance, I will try to send PR about this improvement!

**Comment by szyn** [comment]
@dzannotti PTAL? 🙇

File: digdag-ui/console.jsx [hunk 1]
```diff
@@ -642,6 +642,48 @@ class ProjectsView extends React.Component {
...
 }
 
+class StatusFilter extends React.Component {
+  state = {
+    selectedStatus: 'All'
+  };
+
+  filterSessionsByStatus(sessions, selectedStatus) {
+    switch (selectedStatus) {
+      case 'Success':
```
Comment (dzannotti) [inline digdag-ui/console.jsx]: this cases should use a common constant with the rendering function, so strings can't go out of sync (also we should add a type so flow can ensure consitency)

**Pull request merged**